getMarkers.php

```diff
@@ -273,13 +274,18 @@
 		else	
 			$hippo_color[$name_markers[$f1]] = $img[1];
 	}
-	
 		
 	preg_match('!\d+!',substr($type->getName(),strpos($type->getName(), ')')),$matches);
 	$neurite_pattern=str_split($matches[0]);
-	$neurite_pattern_temp = str_replace($neurite_pattern[$soma_position], "<strong>".$neurite_pattern[$soma_position]."</strong>", $neurite_pattern);
```

Comment:
In this line it used to replace all 2 instead of the value @ 2 position..

in the loop we are looping thru and if the i value is in the soma locations ex: 0 in [0, 2, 3] that value will be highlighted and value @ 2 and 3 positions respectively